gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcRawCallableFactory.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2017 Google LLC
```

Comment:
2020

Given it is 3 years older than should be but matches year of `GrpcCallableFactory`, did you crate this class by copying `GrpcCallableFactory` and then removing stuff from it (i.e. this class is a product of removals, not product of additions)? 

If yes, it makes me feel more confident that there were no typos and missed things in this implementation =)